src/modules/whats-new/components/whats-new-modal.tsx

```diff
@@ -57,6 +57,14 @@ const PageContent = ( {
 export default function WhatsNewModal( { showModal, onClose }: WhatsNewModalProps ) {
 	const locale = useI18nLocale();
 	const whatsNewPages: WhatsNewPage[] = [
+		{
+			image: selectiveSyncIllustration,
+			title: __( 'Synchronize with precision' ),
+			description: __(
+				'Choose the parts of your site to sync, speeding up the process and reducing unnecessary changes.'
```

Comment:
@gavande1, Let's apply Wojtek's feedback and use `synchronize` instead of `sync`.

```suggestion
				'Choose the parts of your site to synchronize, speeding up the process and reducing unnecessary changes.'
```

Reply:
Thanks for highlighting that. I have addressed in [1413421](https://github.com/Automattic/studio/pull/1520/commits/1413421f6309c360caac007946bee9d9f8ca3611).